packages/calypso-e2e/src/lib/utils/criteria-for-test-accounts.ts

```diff
@@ -56,6 +56,15 @@ const defaultCriteria: FeatureCriteria[] = [
 		siteType: 'atomic',
 		accountName: 'gutenbergAtomicSiteEdgeUser',
 	},
+
+	// @todo consider adding a special '*' wildcard value to ignore certain
```

Comment:
Ooo, that would be really nice and clean this up a good bit! I agree it can wait, but definitely a cool idea!